WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -571,4 +571,79 @@
       end
     end
   end
+
+  describe 'GET #my_competitions' do
```

Comment:
Thanks @larspetrus. I'll create a test without a start_date and clean up the tests as you suggested.